src/riak_test_escript.erl

```diff
@@ -148,9 +148,12 @@ main(Args) ->
                     Tests0;
                 {_, undefined} ->
                     Tests0;
+                %% TODO don't understand what this does
```

Comment:
Did you understand Jon's explanation?  If so, we can probably remove this comment.